dynamic_programming/fizz_buzz.py

```diff
@@ -0,0 +1,61 @@
+# https://en.wikipedia.org/wiki/Fizz_buzz#Programming
+
+
+def fizz_buzz(number: int, out: str, iterations: int) -> str:
```

Comment:
Why are you passing `out` as a parameter?

It's a temp variable an will always be `""`

Reply:
okay ill fix